xbmc/pvr/channels/PVRChannelGroup.cpp

```diff
@@ -846,7 +846,9 @@ bool CPVRChannelGroup::Renumber(void)
       (*it).channelNumber = currentChannelNumber;
     }
 
-    (*it).channel->SetChannelNumber((*it).channelNumber);
+    // TODO: This is a quick fix for v18. Whole channel number handling should be reworked - code is imo unmaintainable.
```

Comment:
make this a doxygen todo
```suggestion
    //! @todo This is a quick fix for v18. Whole channel number handling should be reworked - code is imo unmaintainable.
```

Reply:
will do. thx